src/components/activity-page/managed-interactive/managed-interactive.tsx

```diff
@@ -175,6 +177,42 @@ export const ManagedInteractive: React.ForwardRefExoticComponent<IProps> = forwa
     setActiveLightbox(null);
   };
 
+  const handleGetAttachmentUrl = async (request: IAttachmentUrlRequest): Promise<IAttachmentUrlResponse> => {
+    const { name, operation, type, expires, requestId } = request as any; // TODO: remove cast once next version of library is published
```

Comment:
@kswenson perhaps you have this change locally and having pushed it, but I'd expect this `type` to be `contentType` based on the update IAttachmentUrlRequest.

Reply:
Yes, local change to be pushed shortly.